experimental/sdata/timeseries/wide.cue

```diff
@@ -0,0 +1,31 @@
+package timeseries 
+
+import sdata "github.com/grafana/grafana-plugin-sdk-go/experimental/sdata"
+
+#TimeField: sdata.#FieldSchema & {
```

Comment:
should all of these definitions include a `#` prefix? it seems like cue would look for these in the json that was being validated unless i added the prefix

Reply:
If you want the field to not be a part of what's looked for e.g. in validation, then yes, a definition (`#`-prefixed field) will do it. But using definitions also means that anywhere the field is reference, the schema is closed (no add'l fields allowed). That's probably the behavior you want here, but if you don't, you may be able to use a hidden field instead - `_`-prefixed.

This is an area where i think CUE may have smashed a few too many behaviors into too few operators (regular fields vs. definitions vs. hidden)